platform/ios/ios.xcodeproj/project.pbxproj

```diff
@@ -2609,6 +2621,7 @@
 					"$(geometry_cflags)",
 					"$(geojson_cflags)",
 				);
+				OTHER_SWIFT_FLAGS = "-warnings-as-errors";
```

Comment:
Clever idea. I'm a bit concerned that this could be problematic in the future, since Swift doesn't have the pragmas that Objective-C does for intentionally suppressing warnings. On the other hand, if we rely on per-file flags, we risk missing the flag on a test file. So let's go with this and consider pulling it out if we ever need to live with a warning without porting a test to Objective-C.